server/app/app.py

```diff
@@ -44,10 +44,11 @@
 
 
 def run_scanpy(args):
+    host = "0.0.0.0"  # Necessary for redirection
```

Comment:
Can you pull the changes from master for app.py? I think one of our other PRs have solved this problem.